modules/data.remote/DESCRIPTION

```diff
@@ -36,4 +36,4 @@ Copyright: Authors
 LazyLoad: yes
 LazyData: FALSE
 Encoding: UTF-8
-RoxygenNote: 7.0.2
+RoxygenNote: 7.1.0
```

Comment:
@infotroph sorry for bothering you about this again, but we are still at roxygen `7.0.2` rgiht?

Reply:
We are but it may be time to change that. If someone wants to take it on before I can, the process is basically:

* Pick a date for the update and warn everyone
*Iideally close out as many open PRs as possible before then to minimize merge conflicts
* Install the newest Roxygen on your dev machine
* git pull, make sure your tree is clean, then `make clean && make document`
* Commit all the changed Rd and DESCRIPTION files
* `grep -R 7.0.2 .` -> review, update appropriately, commit. I think at least three scripts `scripts/travis/install.sh`, `.github/workflows/styler-actions.yml`, and `docker/depends/Dockerfile` will need updating, but I may be forgetting others.